### PR TITLE
Accessibility - pair programming 😎👁

### PR DESCRIPTION
**Pull request by rotimiazeez**
# Portfolio Page: Accessibility 🎉 

> This PR describes adding web accessibility in accordance with the [W3C](https://www.w3.org/). 💪🏽 

Additional description of the project and its features. ➕ 

- Page title - Changed to "Azeez's portfolio" (which describe the page).
- Image text alternatives** (alt text) - No issues found.
- Text headings - No issues found.
- Color contrast - Colors fixed.
- Resize - No issues found.
- Interaction - Interactions fixed for a better experience.
- Moving content - No issues found.
- Multimedia - No issues found.
- The basic structure of the page - No issues found.

## Built With ❤️ 

- HTML
- CSS

## Deployed Preview 🚀 
[Deploy link](https://deploy-preview-8--rotimiazeez-portfolio.netlify.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/style.css">
-    <title>Hello world</title>
+    <title>My Portfolio</title>
 </head>
```
Comment (Emmyn5600) [inline index.html:10]: - [ ] Kindly, use better page titles:
      For instance: Your name portfolio (Rotimiazeez's Portfolio)

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @rotimiazeez 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:

You need to make more commits by changing the necessary changes :adult: 
  
- [x] Kindly, follow the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)
- [x] Please double check the color contrast of the project. Use this [link](https://www.aditus.io/button-contrast-checker/) to check for the color contrast and fix it. 
- [x] Some of the colors you've chosen for your portfolio currently don't meet WebAIM accessibility standards for contrast ratio. Please readjust your color palette by using a color contrast validator tool like this [one](https://color.a11y.com/%5C)  to access contrast ratio.
- [ ] Kindly, work on the work section to change the font-weight of the text and add hover color to your `see project` button. 

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (rotimiazeez) [reply index.html]: 👍🏽

**Comment by rotimiazeez** [comment]
Hi @Emmyn5600 👋🏽, thanks for pointing out the changes in the suggestion 👏🏽, changes have been made and I hope it gets approved this time. 💪🏽

File: index.html [hunk 2]
```diff
@@ -88,14 +88,14 @@ <h4 class="cards-text-header bold font-size-32 line-height-44 black pad-t-8 slat
...
                         <li
-                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
+                            class="font-size-13 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
                             css
                         </li>
                         <li
-                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
+                            class="font-size-13 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
                             javascript</li>
                     </ul>
```
Comment (Emmyn5600) [inline index.html:100]: - Only two issues left to get approved:

- [ ] Kindly, work on Get my resume button to look more attractive, I suggest to do it as you did for `see project` button 
- [ ] Arrange JavaScript button well in the same line as you did for both HTML and CSS.

For more details:
Selection_107

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @rotimiazeez 

Good job so far!:clap::clap:
You have done well in implementing the changes requested by the previous code reviewer  :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 2 above)
Comment (rotimiazeez) [reply index.html:100]: Done 👍🏽

**Comment by rotimiazeez** [comment]
Hi @Emmyn5600 👋🏽, thanks for pointing these out. I wish you had added it to the first changes, it would have saved me from the second attempt 🤷🏽. Changes have been made and I hope it gets approved this time. 💪🏽

**Comment by rloterh** [comment]
Issues well resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Azeez and Carlos, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾👍.
Congratulations! 🎉🥂

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
@rloterh, thanks for the approval 👍🏽 🎉.

**Pull request merged**